app/javascript/App.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import Greetings from './Greetings'
+
+function App() {
+  return <Greetings />
+}
+
```

Comment:
- [ ] Based on the requirements, kindly make sure that your  `App` component uses routes (from react-router)

Reply:
@prolajumokeoni Thank you, I fixed it.